install/configuring_inventory_file.adoc

```diff
@@ -217,6 +217,13 @@ xref:configuring-node-host-labels[Configuring Node Host Labels] for details.
 |This variable enables the template service broker by specifying one or more
 namespaces whose templates will be served by the broker.
 
+|`openshift_master_bootstrap_auto_approve`
```

Comment:
@sdodson, @frobware,  I added the parameter to the General Cluster Variables table. Please let me know if you see more changes.